saltychat/SaltyShared/Event.cs

```diff
@@ -34,5 +34,9 @@ public class Event
         public const string SaltyChat_SetRadioChannel = "SaltyChat_SetRadioChannel";
         public const string SaltyChat_UpdateRadioTowers = "SaltyChat_UpdateRadioTowers";
         #endregion
+
+        #region Megaphone
+        public const string SaltyChat_IsSpeaking = "SaltyChat_IsSpeaking";
```

Comment:
Would think of a new name, since `IsSpeaking` could irritate others and they think that this event triggers, if you just start talking.

Reply:
Thinking of another name: `IsUsingMegaphone` ?

Reply:
Sure, that works.